### PR TITLE
Docs/auth linkedin notion

### PR DESCRIPTION
**Pull request by burggraf**
## What kind of change does this PR introduce?

Adds auth guide for LinkedIn OAuth

### CODE REVIEW

**Comment by Isaiah-Hamilton** [comment]
@burggraf can you add LinkedIn to this [list of auth providers](https://github.com/supabase/supabase/blob/master/web/src/data/authProviders.js).

**Comment by burggraf** [comment]
> @burggraf can you add LinkedIn to this [list of auth providers](https://github.com/supabase/supabase/blob/master/web/src/data/authProviders.js).

Done.

**Review by Isaiah-Hamilton — APPROVED** [review]
lgtm

File: web/docs/guides/auth/auth-linkedin.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+</video>
+
+### Create a LinkedIn OAuth app
+
+- Go to [LinkedIn Developer Dashboard](https://www.linkedin.com/developers/apps).
+- Click on `Create App` at the top right
+- Enter your `LinkedIn Page` and `App Logo`
+- Save your app
+- Click `Auth` from the top menu
+- Add yuor `Redirect URL` to the `Authorized Redirect URLs for your app` section
```
Comment (hellno) [inline web/docs/guides/auth/auth-linkedin.mdx:52]:
```suggestion
- Add your `Redirect URL` to the `Authorized Redirect URLs for your app` section
```

**Review by hellno — COMMENTED** [review]
found this tiny typo :)

Do you have any update on when this is getting merged? Would love to test LinkedIn as auth provider for supabase

**Pull request merged**